src/ghdcbot/__init__.py

```
@@ -0,0 +1 @@
+"""Discord–GitHub automation engine."""
```

Ruff (0.14.13)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)